test/unit/sorted-views.spec.js

```diff
@@ -391,4 +391,85 @@ describe('collection/composite view sorting', function() {
       });
     });
   });
+
+  describe('when using `{ reorderOnSort: true }`', function () {
+    var texts = {
+      asOption: 'as an option',
+      onPrototype: 'on the prototype',
+      viewComparator: 'with viewComparator'
+    };
+
+    var getSpecTitle = function(options) {
```

Comment:
`getSpecTitle` gets an object because the `specOptions` is an object meant to imitate a `set`. I could do `getSpecTitle(_.keys(specOptions))` and have:
```
getSpecTitle: function (options) {
   return _.map(options, function (k) { return texts[k]; })
}
```

but I do not think its better. Anyway, at this point, it really is a matter of choice.